notebooks/finetune/Flan-T5-3B/RosaQA.ipynb

```diff
@@ -0,0 +1,7957 @@
+{
```

Comment:
<p>Should we show a "before" and "after" version?</p><p>Here we have the validation "after" the training, but it could be interesting to also show the output from the same questions from the untrained module, for comparison.</p><p> </p><p>Also: we are not setting any context for the questions. That's fine, but maybe add a short note about that? why are we including "context" in the prompt (and fine tuning process) if we are not using it? How should the context look like when/if used?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Added the "before" and "after" version. </p><p> </p><p>For the context comment, the validation data doesn't have any so I'm leaving it for now, for fine tuning the context is in the dataset, I've added the head of the dataset for reference. Ideally, for validation we should have embedding search context. That's the next to do. </p><p> </p>